source/_integrations/iotty.markdown

```diff
@@ -34,15 +34,23 @@ This integration currently supports the following iotty devices:
   - [2-Switch Controller](https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925)
   - [3-Switch Controller](https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269)
   - [4-Switch Controller](https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901)
-  EU version:
+  EU version: \
   - [iotty Smart Switch](https://iotty.uk/collections/frontpage/products/iotty-smart-switch)
   - [iotty Smart Switch (variant)](https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082)
   - [iotty Plus Interruttore Intelligente](https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli)
 
+- iotty Shutter \
+  - [iotty Smart Shades Switch](https://iotty.uk/collections/frontpage/products/e2s-plus-smart-shades-switch-for-shutters-and-blinds)
+  - [iotty Plus Interruttore Intelligente per Tende e Tapparelle](https://iotty.it/collections/prodotti-singoli/products/i3s-plus-interruttore-intelligente-per-tende-e-tapparelle)
```

Comment:
our documentation is in English. Is there an English equivalent to this page?

Reply:
Hi, these are the listing of the products, the first link you see here is for the whole EU marked, and thus is in english, the second one is only for Italian market, so there are no other languages available at the moment.